src/spetlrtools/diagrams/LibraryParser.py

```diff
@@ -0,0 +1,80 @@
+import importlib
+import inspect
+import pkgutil
+import re
+from textwrap import dedent
+
+import yaml
+
+from spetlrtools.diagrams.Edge import Edge
+
+
+class LibraryParser:
```

Comment:
Give a more spetlr like name? 

LibraryParser is very generic. 

Like: SpetlrDiagramLibraryParser. Maybe you can find a more readable name